Header header
float32 x1
float32 x2
float32 e
float32 ref_y